### PR TITLE
Enable intersphinx

### PR DESCRIPTION
**Pull request by chmelevskij**
If referecing Django settings following syntax can be used:
:django:setting:`INSTALLED_APPS`

### CODE REVIEW

File: docs/content-architecture.rst [hunk 1]
```diff
@@ -269,12 +269,12 @@ custom :class:`.Page` models and are then called inside the
 :class:`.Page` instance. A Page Processor will always be passed two arguments
 - the request and the :class:`.Page` instance, and can either return a
 dictionary that will be added to the template context, or it can return
-any of Django's ``HttpResponse`` classes which will override the
+any of Django's `HttpResponse`` classes which will override the
```
Comment (stephenmcd) [inline docs/content-architecture.rst]: This looks wrong

**Comment by stephenmcd** [comment]
Cool idea! Could you apply it to the rest of the documentation?

File: docs/content-architecture.rst (hunk 1 above)
Comment (chmelevskij) [inline docs/content-architecture.rst]: It is wrong... Probably just something left over from testing.

**Comment by chmelevskij** [comment]
Will do that as well. Should I just close this pull request and make a new one afterwards? I just thought it could be useful for someone writing the docs as well.

**Comment by stephenmcd** [comment]
You could just add to this current PR if you like. Thanks a lot!

**Comment by pedromorgan** [comment]
I was about to make an issue of interphinx, but its already here

**Comment by chmelevskij** [comment]
All that needs to be done is to add the actual tags.....

**Pull request closed without merging**